app/config.py

```diff
@@ -11,58 +11,61 @@ class Config(object):
 
     CHECK_PROXY_HEADER = False
 
-    DANGEROUS_SALT = os.environ.get('DANGEROUS_SALT')
-
     # Logging
     DEBUG = False
 
     DOCUMENT_DOWNLOAD_API_HOST_NAME = os.environ.get('DOCUMENT_DOWNLOAD_API_HOST_NAME')
-    DOCUMENT_DOWNLOAD_ENVIRONMENT = 'development'
-    DOCUMENT_DOWNLOAD_LOG_PATH = os.getenv('DOCUMENT_DOWNLOAD_LOG_PATH')
 
     HEADER_COLOUR = '#FFBF47'  # $yellow
     HTTP_PROTOCOL = 'http'
 
     ROUTE_SECRET_KEY_1 = os.environ.get('ROUTE_SECRET_KEY_1', '')
 
-    SHOW_STYLEGUIDE = True
-
+    # needs to refer to notify for utils
     NOTIFY_LOG_PATH = '/home/vcap/logs/app.log'
 
 
 class Development(Config):
+    API_HOST_NAME = 'http://localhost:6011'
+    ADMIN_BASE_URL = 'http://localhost:6012'
```

Comment:
How is this used?

Reply:
this is used to get the CDN domain, to get logos etc from